Lib/contextlib.py

```diff
@@ -88,7 +88,7 @@ def __exit__(self, type, value, traceback):
             try:
                 next(self.gen)
             except StopIteration:
-                return
+                return None
```

Comment:
For consistency with the other return statements, this should be `return False`.